backend/de.metas.adempiere.adempiere/base/src/main/java/de/metas/externalsystem/ExternalSystemRepository.java

```diff
@@ -68,19 +56,19 @@ private ExternalSystemMap retrieveMap()
 				.collect(ExternalSystemMap.collect());
 	}
 
-	public ExternalSystem getByValue(@NonNull final ExternalSystemType value)
+	public ExternalSystem getByType(@NonNull final ExternalSystemType type)
```

Comment:
@mrie04 not sure if now, but in future we might have multiple external systems for a type.

e.g. the "scripted" type might have couple of external systems defined with different configs.

Reply:
@teosarca no because type contains String of ExternalSystem.value and this column has an unique index